<launch>
  <node name="vfhp_planner" pkg="vfhp_local_planner" type="vfhp_node.py" output="screen">
    <param name="odom_frame_id"   type="str"     value="odom"/>
    <param name="robot_frame_id"   type="str"     value="mecanum_base"/>
    <param name="decay_rate"       type="int"     value="40"/>
    <param name="decay_value"      type="int"     value="2"/>
    <param name="decay_guardband"  type="int"     value="10"/>
    <param name="grid_size"        type="int"     value="325"/>
    <param name="c_max"            type="int"     value="20"/>
    <param name="resolution"       type="double"  value="0.12"/>
    <param name="window_size"      type="int"     value="29"/>
    <param name="hist_size"        type="int"     value="180"/>
    <param name="robot_radius"     type="double"  value="0.498"/>
    <param name="d_s"              type="double"  value="0.10"/>
    <param name="kb"               type="double"  value="45"/>
    <param name="t_lo"             type="double"  value="355000.0"/>
    <param name="t_hi"             type="double"  value="450000.0"/>
    <param name="v_max"            type="double"  value="0.31"/>
    <param name="v_min"            type="double"  value="0.0"/>
    <param name="mu1"              type="double"  value="6.0"/>
    <param name="mu2"              type="double"  value="2.0"/>
    <param name="mu3"              type="double"  value="2.0"/>
  </node>
</launch>
